src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -70,12 +70,16 @@ public BidRequest fillImplicitParameters(BidRequest bidRequest, RoutingContext c
         final Device populatedDevice = populateDevice(bidRequest.getDevice(), request);
         final Site populatedSite = bidRequest.getApp() == null ? populateSite(bidRequest.getSite(), request) : null;
         final User populatedUser = populateUser(bidRequest.getUser(), context);
+        final Integer at = bidRequest.getAt();
 
-        if (populatedDevice != null || populatedSite != null || populatedUser != null) {
+        if (populatedDevice != null || populatedSite != null || populatedUser != null || at == null || at == 0) {
```

Comment:
`at == null || at == 0` condition appears twice in this method and could be extracted to a variable